spec/index.html

```diff
@@ -1152,6 +1152,22 @@ <h3>The XML Schema Built-in Datatypes</h3>
         model.</li>
     </ul>
 
+    <p class="note">The <a>value spaces</a>
+      of <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a> and
+      <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a> do not
+      include all decimal numbers. For every literal of
+      any of these two datatypes, the value
+      of the literal is a value that can be represented as an
+      <a href="http://ieeexplore.ieee.org/servlet/opac?punumber=4610933">IEEE 754-2008</a>
+      binary floating point representation of the corresponding precision.
+      For instance, the literal with
+      lexical form `"0.1"` and datatype <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a>
+      <a>denotes</a> the number 0.100000001490116119384765625.
+      As an alternative to <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a>
+      and <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a>, the
```

Comment:
```suggestion
      Rather than <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a>
      or <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a>, the
```